tests/js/lib/errors.js

```diff
@@ -0,0 +1,17 @@
+// Simple function to examine validation errors.
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+var jqUnit = require("jqUnit");
+
+require("../../../src/js/common/validate");
+
+fluid.registerNamespace("gpii.schema.tests");
+gpii.schema.tests.hasFieldErrors = function (results, fields) {
+    fluid.each(fields, function (field) {
+        var path = typeof field === "string" ? [field] : field;
+        var target = gpii.schema.validator.resolveTargetFromPath(results.fieldErrors, path);
```

Comment:
I renamed the function to make it clearer that it would create paths and added documentation.  I hope this will make you comfortable, as we need to ensure that an array already exists at the desired path when registering new errors.